components/gnome-connection-manager/gnome-connection-manager.p5m

```diff
@@ -0,0 +1,54 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2014 Alexander Pyhalov. All rights reserved.
+# Copyright (c) 2015 Jim Klimov. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# Add-on for this package
+file path=usr/bin/gcm
+# Files delivered by source tarball, as of version 1.1.0, listed via:
+#   find . -type f | sed 's,^./,file path=usr/share/gnome-connection-manager/,'
+file path=usr/share/gnome-connection-manager/donate.gif
+file path=usr/share/gnome-connection-manager/gnome_connection_manager.py
```

Comment:
I tried mode=755 and the make process failed because pkgdepend's python inspector could not determine which version of Python to use - no explicit filename in shebang, and no hints in the code. I'm not a Python coder, so have little input on that, except possibly providing a patch to change the shebang to an explicit version. Currently I went the other way, keeping the script non-executable and calling `/usr/bin/python /path/to/script` from the `/usr/bin/gcm` wrapper.

The GCM project states (via README.txt in the tarball) that it needs "python-2.6 or newer". I'm not sure it is nice to explicitly pull in a dependency on the older version which we might want to get rid of soon, when 2.7 is also available in Hipster repos, and 3.x are in upstream. Currently I'm not at the PC where the work was done, but I think it has both python-26 and python-27 runtimes installed, and whatever is default via `/usr/bin/python` - suffices to run the script there.